pilot/pkg/model/extensions.go

```diff
@@ -56,6 +92,49 @@ type WasmPluginWrapper struct {
 	WasmExtensionConfig *envoyWasmFilterV3.Wasm
 }
 
+func (p *WasmPluginWrapper) ShouldApplyTo(proxyLabels map[string]string, li *WasmPluginListenerInfo) bool {
```

Comment:
I do think the `ShouldApplyTo` is good name for this, prefer to something like `Match` or `MatchListener`?

Reply:
MatchListener seems to be a better name.
Changed.

Reply:
Let's go with `MatchListener`. It seems to be more consistent with the names of the sub-functions.

Done.